lib/exercism/user_statistics.rb

```diff
@@ -0,0 +1,30 @@
+class UserStatistics
+  def self.of(username)
+    by_username(username)
+  rescue Exception => e
+    Bugsnag.notify(e, {username: username})
+    id
+  end
+
+  def self.by_username(username)
+    user = User.find_by(username: username)
+    { user: user.public_user_attributes, 
```

Comment:
I might be wrong but I think it's a simple as having a space before the end of line here on L11.

`! git grep ' $' -- \*.rb` is something like grep in files that end in .rb that have spaces before the end of line (`' $'`).